watershed_workflow/densify_rivers_hucs.py

```diff
@@ -0,0 +1,278 @@
+"""this code increases the resolution of river network and huc boundmary in a controlled way using original river network and huc"""
+import logging
+import numpy as np
+import math
+from scipy import interpolate
+import shapely
+
+import watershed_workflow.utils
+
+def densify_rivers(rivers,rivers_=None, use_original=False, limit=100, treat_collinearity=False):
+    """Returns a list for densified rivers"""
+    rivers=[densify_river(river, river_, use_original= use_original, limit=limit, treat_collinearity=treat_collinearity) for river, river_ in zip(rivers, rivers_)]
+    return rivers
+
+def densify_river(river,river_=None, use_original=False, limit=100, treat_collinearity=False):
```

Comment:
Change river_ to something else.  Trailing underscores often mean something specific (at least in C++) and could lead to confusion.

Maybe "raw_river" or "river_raw"?

Reply:
Also node_